src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -84,14 +86,14 @@ const Body = BaseFormWithPolling.extend(Object.assign(
           model.trigger('clearFormError');
         }, 0);
         setTimeout(() => {
-          this.startPolling();
+          this.startPolling(currentRefreshInterval);
```

Comment:
if  `this.dynamicPollingInterval` has value.. it takes precedence
https://github.com/okta/okta-signin-widget/blob/f6225f4320e631e4a852c843012d78f9f43baf60/src/v2/view-builder/views/shared/polling.js#L42

Reply:
A thought - Different variables are being used to make decisions in different places here which makes it confusing and hard to maintain. How about adding another flag like `forceRemediationPolling` default to `false`:
```
startPolling(newRefreshInterval, forceRemediationPolling = false) {
  .
  .
  if (this.fixedPollingInterval || forceRemediationPolling) { 
     this._startRemediationPolling(); 
  }
  .
  .
}
```

and line 89 can be -
```
this.startPolling(currentRefreshInterval, true);
```

* Our decision to call `_startRemediationPolling` is not based off of `fixedPollingInterval` and then actually not use it later.
* Guard any changes to `this.options.currentViewState.refresh` and make sure `remediationPolling` is always called in this case.

Reply:
@anipendakur-okta 
I have simplified the function call from `BaseAuthenticatorEmailView.js` to just `this.startPolling(this.options.appState.get('dynamicRefreshInterval'));` since the function already handle the fallback to `fixedPollingInterval` in case `this.options.appState.get('dynamicRefreshInterval')` is `undefined`.

This should be simpler because `newRefreshInterval` will always take precent over `fixedPollingInterval` with a fallback. The caller doesn't have to handle different variable to decide. `newRefreshInterval` take the first priority.

Reply:
So much better now, lgtm!